features/fields.feature

```diff
@@ -0,0 +1,341 @@
+      | naam                             | waarde    |
+      | burgerservicenummer              | 999995078 |
+      | geslachtsaanduiding.code         | M         |
+      | geslachtsaanduiding.omschrijving | man       |
+
+    Scenario: Gevraagde velden zijn gegevens van één gegevengroep van de persoon
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                                                |
+      | type                | RaadpleegMetBurgerservicenummer                       |
+      | burgerservicenummer | 999995078                                             |
+      | fields              | burgerservicenummer,naam.geslachtsnaam,naam.voornamen |
+      Dan heeft de persoon met burgerservicenummer '999995078' alleen de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 999995078 |
+      En heeft de persoon met burgerservicenummer '999995078' alleen de volgende 'naam' gegevens
+      | naam          | waarde   |
+      | geslachtsnaam | Groenen  |
+      | voornamen     | Franklin |
+
+    Scenario: Gevraagde velden zijn gegevens van meerdere gegevensgroepen van de persoon
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                                                          |
+      | type                | RaadpleegMetBurgerservicenummer                                 |
+      | burgerservicenummer | 999995078                                                       |
+      | fields              | burgerservicenummer,geslachtsaanduiding.code,naam.geslachtsnaam |
+      Dan heeft de persoon met burgerservicenummer '999995078' alleen de volgende gegevens
+      | naam                     | waarde    |
+      | burgerservicenummer      | 999995078 |
+      | geslachtsaanduiding.code | M         |
+      En heeft de persoon met burgerservicenummer '999995078' alleen de volgende 'naam' gegevens
+      | naam          | waarde  |
+      | geslachtsnaam | Groenen |
+
+    Scenario: Gevraagd veld is een gegeven van een gegevensgroep collectie van de persoon
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                                                         |
+      | type                | RaadpleegMetBurgerservicenummer                                |
+      | burgerservicenummer | 999995078                                                      |
+      | fields              | burgerservicenummer,partners.type,partners.burgerservicenummer |
+      Dan heeft de persoon met burgerservicenummer '999995078' alleen de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 999995078 |
+      En heeft de persoon met burgerservicenummer '999995078' een 'partner' met alleen de volgende gegevens
+      | naam                | waarde    |
+      | type                | Partner   |

```

Comment:
Moet dit 'type' veld niet toegevoegd worden aan de tabel die bovenaan dit bestand bij 'Achtergrond' wordt vermeld?

Reply:
type wordt afgeleid (zie partners.feature), dus kan niet in Gegeven staan

Reply:
Ok